packages/eslint-config/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Shared ESLint configuration",
   "private": true,
   "devDependencies": {
+    "@shared/prettier-config": "file:../prettier-config",
```

Comment:
why do we need add this ref? seems circulate ref between eslint-confg and prettier-config